lightkurve/lightcurve.py

```diff
@@ -1919,6 +1919,66 @@ def plot_river(self, period, epoch_time=None, ax=None, bin_points=1,
             ax.set_aspect(a/b)
         return ax
 
+    def create_transit_mask(self, period, transit_time, duration):
+        """Returns a np.array that is False during transits and True everywhere else.
+
+        This function is flexible to multi-planet systems, allowing users to pass
+        in transit parameters as arrays or lists.
+
+        Parameters
+        ----------
+        period : float, or array-like
+            Period of the transits.
+        duration : float, or array-like
+            Duration of the transits.
+        transit_time : float, or array-like
+            Transit midpoint of the transits.
+
+        Returns
+        -------
+        transit_mask : np.array of Bool
+            Mask that removes transits. Mask is True where there are no transits.
+        """
+
+        # Make sure all params have the same number of entries
+        n_planets = len(np.atleast_1d(period))
+        if any(len(param) != n_planets for param in [np.atleast_1d(transit_time),
+                                                     np.atleast_1d(duration)]):
+            raise ValueError("period, duration, and transit_time must have "
+                             "the same number of values.")
+
+        # Create empty cadence mask
+        transit_mask = np.empty(len(self), dtype=bool)
+        transit_mask[:] = True
+
+        # Create a transit mask for a single planet
+        if isinstance(period, (float, int)):
+            in_transit = self._get_masked_cadences(period, duration, transit_time)
```

Comment:
I recommend ensuring that period/duration/transit_time is always an array (e.g. using np.atleast_1d at the start of a function), and treating the single-planet case as a "multi-planet system with number of planets = 1".  Put differently, I recommend getting rid of lines 1954-1959 and always treat period/duration/transit as a list of size 1 or longer.